tasks/main.yml

```diff
@@ -85,10 +85,16 @@
       validate:
         fail_on_error: yes
         strict: yes
-    register: deploy_account
+
+  - name: Get deploy account name
+    k8s:
```

Comment:
I think we want `k8s_info` here, which will query an existing object. `k8s` will actually attempt to re-apply the definition.